### ISSUE TITLE
Mint on Solana Blockchain

### ISSUE THREAD

**Issue by lukaswolfrum** [amosproj/amos2022ss07-nft-playbook#5]
## User story
1. As a power user
2. I want / need to mint an NFT on the Solana Blockchain
3. So that I can easily include the NFTs in my project

## Acceptance criteria
* Criterion 1
* Criterion 2
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
